src/languageservice/services/yamlCodeLens.ts

```diff
@@ -93,7 +94,7 @@ function addSchemasForOf(schema: JSONSchema, result: Map<string, JSONSchema>): v
 
 function addInnerSchemaUrls(schemas: JSONSchemaRef[], result: Map<string, JSONSchema>): void {
   for (const subSchema of schemas) {
-    if (isJSONSchema(subSchema)) {
+    if (!isBoolean(subSchema)) {
```

Comment:
This still does not confirm that it is a JSONSchema.

"I'm looking for a dog. Is a bird? No, it's not a bird. OK, it's not a bird so it must be a dog" That does not work.

Reply:
We have only two choices, `subSchema` can be boolean or JSONSchema
It would be enough to check that schema is not boolean, as rest check will do TS compiler.

Reply:
OK, since you are not using an `any` I guess that could work. 

I was thinking that if you wanted to confirm that matched the interface then test for expected mandatory fields. But I'll leave it up to you.